test-framework/junit5/src/main/java/io/quarkus/test/junit/IntegrationTestUtil.java

```diff
@@ -232,8 +232,23 @@ static Properties readQuarkusArtifactProperties(ExtensionContext context) {
         Path buildOutputDirectory = determineBuildOutputDirectory(context);
         Path artifactProperties = buildOutputDirectory.resolve("quarkus-artifact.properties");
         if (!Files.exists(artifactProperties)) {
-            throw new IllegalStateException(
-                    "Unable to locate the artifact metadata file created that must be created by Quarkus in order to run integration tests.");
+            TestLauncher testLauncher = determineTestLauncher();
+            String errorMessage = "Unable to locate the artifact metadata file created that must be created by Quarkus in order to run integration tests. ";
+            if (testLauncher == TestLauncher.MAVEN) {
+                errorMessage += "Make sure this test is run after 'mvn package'. ";
+                if (context.getTestClass().isPresent()) {
+                    String testClassName = context.getTestClass().get().getName();
+                    if (testClassName.endsWith("Test")) {
+                        errorMessage += "The easiest way to ensure this is by renaming the test to '"
+                                + (testClassName.substring(0, testClassName.length() - 4) + "ΙΤ'.");
```

Comment:
This advice expects failsafe to be configured already, right?
Maybe just add ` to be run by maven-faislafe-plugin.`?

Reply:
Good point. I'll follow your suggestion